### PR TITLE
Assignment backend-1-lesson-1

### PR DESCRIPTION
**Pull request by NataMarkova**
@strom2357 , please review my SQL statements. Thank you in advance.

### CODE REVIEW

File: sql.txt [hunk 1]
```diff
@@ -1,15 +1,54 @@
...
+    	(NULL, 'Product 3', 4, 2, '10 boxes x 30 bags', 45);
+
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Just specify the new value; do 
 not try to get SQL to do the addition for you.)  Paste your SQL statement below
 
+
+UPDATE Products 
+SET Price = Price + 1.5 
+WHERE ProductID BETWEEN 1 AND 2;
```
Comment (andresochoa91) [inline sql.txt:47]: The exercise says we have to specify the value, not adding 1.50 using addition.

**Review by andresochoa91 — CHANGES_REQUESTED** [review]
The exercise looks good. There is just one thing to change (the solution is correct, however, in the exercise 5 we should follow the instructions).

File: sql.txt (hunk 1 above)
Comment (andresochoa91) [inline sql.txt:47]: The exercise says we have to specify the value, not adding 1.50 using addition.

**Review by andresochoa91 — CHANGES_REQUESTED** [review]

File: sql.txt (hunk 1 above)
Comment (NataMarkova) [reply sql.txt:47]: @andresochoa91, yes you right, but John McGarvey slaked in sunrise-piranha channel and said that there is a bug in this task, so I can't do updating two rows in one statement without having SQL do the addition for me.

Comment (andresochoa91) [reply sql.txt:47]: Oh, okay, then, no worries :+1:

**Pull request merged**